app/book/print.py

```diff
@@ -0,0 +1,29 @@
+from abc import abstractmethod, ABC
+
+from app.book.book import Book
+
+
+class Print(ABC):
+    @abstractmethod
+    def print_book(self, book: Book) -> None:
+        pass
+
+
+class ConsolePrint(Print):
```

Comment:
Rename `ConsolePrint` to `PrintConsole`. The logic is that the class name should describe what it does: it displays to the console. The name should read as "Display" (what it does) followed by "Console" (where it does it).